generate-Lets-Encypt-certificate-using-Concert-for-Minio.md

```diff
@@ -1,53 +1,64 @@
-# Generate Let’s Encrypt certificate using Concert for Minio
+# Generate Let's Encrypt certificate using Concert for Minio
 
-[Let’s Encrypt ](https://letsencrypt.org/)is a new Certificate Authority: It’s free, automated, and open (Public Beta)
+[Let’s Encrypt ](https://letsencrypt.org/) is a new free, automated, and open source, Certificate Authority.
 
 [Concert ](https://docs.minio.io/docs/concert) is a console based certificate generation tool for  Let’s Encrypt. It is open source & one of related project from Minio.
 
-We are going to generate Let's Encypt certificate using Concert followed by consuming it with  Minio server.
+In this recipe, we will generate a Let's Encypt certificate using Concert. This certificate will then be  deployed for use in the Minio server.
 
-## Prerequisites
-* You have golang 1.6+ setup up and running, if not follow [this guide](https://docs.minio.io/docs/install-golang).
-* Port 443 for https is open and  free at time of executing `concert`
-* Concert will require root access while executing because only root is allowed to bind to any port below 1024.
-* You have latest Minio server downloaded, if not please follow Minio Server [Quickstart Guide](doc:minio)
-* We are using ``churchofminio.com`` domain name to generate let's encrypt certs for.
+## 1. Prerequisites
 
-## Install Concert
+* Install Minio Server from [here](https://docs.minio.io/docs/minio ).
+* Install Golang from [here](https://docs.minio.io/docs/how-to-install-golang).
+
+## 2. Dependencies
+* Port 443 for https needs to be open and available at time of executing `concert`.
+* Concert needs root access while executing because only root is allowed to bind to any port below 1024.
+* We will be using our own domain ``churchofminio.com``  as an example in this recipe. Replace with your own domain for your needs.
+
+## 3. Recipe Steps
+
+Step 1: Install concert as shown below.
 ```
 $ go get -u github.com/minio/concert
+
 ```
-## Generate Let's Encrypt cert
+Step 2 : Generate Let's Encrypt cert.
+
 ```
 $ sudo concert gen --dir my-certs admin@churchofminio.com churchofminio.com
 2016/04/04 07:10:01 Generated certificates for churchofminio.com under my-certs will expire in 89 days.
 ```
-Now list your certs saved in `my-certs` directory
+Step 3 : Verify Certificates.
+List certs saved in `my-certs` directory.
+
```

Comment:
We need to give more space between the steps and the next line as in the final view it is showing a continuation. Same  changes need to be done for other steps.